### PR TITLE
#887 Twitter Card support

### PR DESCRIPTION
**Pull request by kevincrafts**

### CODE REVIEW

File: modules/custom/cu_seo_bundle/cu_seo_bundle.info [hunk 1]
```diff
@@ -5,6 +5,8 @@ core = 7.x
 package = CU-Search Engine Optimization
 project_demo_url = http://www.colorado.edu/webcentral/node/684
 dependencies[] = metatag
+dependencies[] = metatag_opengraph
+dependencies[] = metatag_twitter_cards
```
Comment (alexfinnarn) [inline modules/custom/cu_seo_bundle/cu_seo_bundle.info]: I don't see the description tag. I tried adding text to the summary part of the node form as well as manually setting the text to "and stuff..."

Comment (alexfinnarn) [inline modules/custom/cu_seo_bundle/cu_seo_bundle.info]: See https://github.com/CuBoulder/express/issues/887#issuecomment-302202585

**Comment by owenmorrill** [comment]
Metatag description is showing up for me, but if I edit it nothing changes from the default configuration.

If I change the description metatag to say "and stuff..." and save--it still says "University of Colorado Boulder" when the page loads.

**Comment by alexfinnarn** [comment]
I now see no issues with this PR.

**Comment by cathysnider** [comment]
See https://github.com/CuBoulder/express/issues/887#issuecomment-303831261

**Pull request merged**